eng/scripts/Language-Settings.ps1

```diff
@@ -604,6 +604,7 @@ function SetPackageVersion ($PackageName, $Version, $ServiceDirectory, $ReleaseD
   }
   python "$EngDir/versioning/set_versions.py" --build-type $BuildType --new-version $Version --ai $PackageName --gi $GroupId
   python "$EngDir/versioning/update_versions.py" --update-type library --build-type $BuildType --sr
+  python "$EngDir/versioning/update_versions.py" --update-type library --build-type $BuildType --tf $PackageProperties.ReadMePath
```

Comment:
@g2vinay is it only the one README file that needs to be updated?

Reply:
Yeah, primarily the package README.

For rest, we have some edge cases, like azure-identity version is mentioned in Text Analytics readme as well.
But these are not common scenarios, so don't have to be addressed critically.